src/morph/src/Morph.re

```diff
@@ -1,12 +1,17 @@
-let start = (~servers: list(Server.t), ~middlewares=[], handler) => {
+let start =
+    (
+      ~servers: list(Server.t('req_body)),
```

Comment:
I see it stayed this way but it's totally fine for now.

Reply:
I think we should fix this in another PR, it's not really related to the body. And you can run a morph server without this function anyway